model/src/main/kotlin/config/ScanStorageConfiguration.kt

```diff
@@ -115,3 +116,48 @@ data class PostgresStorageConfiguration(
      *       https://jdbc.postgresql.org/documentation/head/connect.html
      */
 ) : ScanStorageConfiguration()
+
+/**
+ * A class to hold the configuration for SW360.
+ */
+data class Sw360StorageConfiguration(
+    /**
+     * The REST API URL of SW360.
+     */
+    val restUrl: String,
+
+    /**
+     * The authentication URL of your SW360 instance.
+     */
+    val authUrl: String,
+
+    /**
+     * The username for the requests to SW360.
+     */
+    val username: String,
+
+    /**
+     * The password of the SW360 user.
+     */
+    val password: String,
+
+    /**
+     * The client ID of the SW360 instance for the two step authentication.
+     */
+    val clientId: String,
+
+    /**
+     * The password of the client ID.
+     */
+    val clientPassword: String,
+
+    /**
+     * The host of the proxy URL if one is in use.
+     */
+    val proxyHost: String? = null,
+
+    /**
+     * THe port of the proxy URL if one is in use.
+     */
+    val proxyPort: Int? = 0
```

Comment:
Does a default port of 0 really make sense? Shouldn't it default to a usable port?